mariadb_skysql/README.md

```diff
@@ -0,0 +1,65 @@
+# Agent Check: MariaDB SkySQL
+
+## Overview
+
+This check monitors [MariaDB SkySQL][1]. SkySQL is a multi-cloud database-as-a-service offering from MariaDB Plc.
+
+Integration with Datadog allows full monitoring of all your MariaDB SkySQL server deployments: Enterprise Server (Single and Replicated), Xpand, Columnstore, as well as MaxScale. Database administrators can track system resource metrics, such as memory, CPU, disk, and network utilization in addition to database specific telemetry such as query performance, connections, threads, cluster, replication status and many others.
+
+
+## Setup
+
+### Installation
+
+To install the MariaDB SkySQL check on your host:
+
+
+1. Install the [developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit)
+ on any machine.
+
+2. Run `ddev release build mariadb_skysql` to build the package.
+
+3. [Download the Datadog Agent][2].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w ./integrations-extras/mariadb_skysql/dist/datadog_mariadb_skysql-0.0.1-py3-none-any.whl`.
```

Comment:
@alopezz Can you confirm this is the correct command to run an install for the integration? I haven't seen integrations-extras in the path before

Reply:
@alopezz bumping this

Reply:
That is not what we expect customers to run, the usual way is described in https://docs.datadoghq.com/agent/guide/use-community-integrations/?tab=agentv721v621#installation.

An example from an existing integration in this repository: https://github.com/DataDog/integrations-extras/blob/36addd6564825f144581ab81e6f52d378571c2c7/gitea/README.md?plain=1#L30-L32

Reply:
@mariadb-BorisBesky Can you update the install command to reflect the above? The integration version should be included in replace of <INTEGRATION_VERSION>.